run_functional_tests.sh

```diff
@@ -35,8 +35,8 @@ node_modules/protractor/bin/protractor protractor.config.js
 TEST_RESULT=$?
 
 # Cleanup webdriver-manager and web app processes
-fuser -k -n tcp 4444
-fuser -k -n tcp 8088
+kill -9 $(lsof -ti tcp:8088)
+kill -9 $(lsof -ti tcp:4444)
```

Comment:
Why is this change needed? And why is the order changed? Previously we killed the process that owns port 8088 first.